Wiki/models/models-aus2200.md

```diff
@@ -0,0 +1,171 @@
+
+### `housekeep_cycle`
+
+This is a standard task in all UM suites that archives logs and cleans working directories. In this suite it is configured to tar the log files and remove the work directory from the previous cycle.
+
+### `convert_to_nc`
+
+This task converts the raw UM output from the previous cycle to netCDF format. It is advised to retain this step, as the converted netCDF output tales up around 1/6th the disk space of the raw UM output. This step can also act as a 'copy output to final location' step. This task is only run when 'Convert output to netCDF' is set to true under 'General Run Options' (`{{CONVERT_TO_NETCDF}}` in `rose-suite.conf`). The final location of the netCDF files is set in the 'Path to converted netCDF files' configuration option (`{{NC_OUTPUT_PATH}}` in `rose-suite.conf`)
+![convert to netcdf](../Images/AUS2200/convert-netcdf.png)
+The netCDF files are created by the `um2netcdf4` python package, which uses `iris` to create the netCDF files. This step runs on 26 sapphire rapids cores (`normalsr` queue) and completes in about 20 minutes.
+
+### `verify_nc`
+This task uses `iris` and `xarray` to verify that the netCDF files created in the `convert_to_nc` step are correct. This step runs on one Sapphire Rapids node (`normalsr` queue, 104 cpus) and completes in about 10 minutes.
+
+## Restart and Spinup
+
+There are two different run styles for the AUS2200 suite used in experiments, they are the 'continuous' and 'restart' style runs. The continuous run style is fairly straight forward. All that is necessary to configure is the `INITIAL_CYCLE_POINT` and `FINAL_CYCLE_POINT`, and the model will run in 6-hour cycle points between those two time stamps. 
+
+Restart style runs are more complex, as each restart may require spin-up time, and output fields may need to be carried over between overlapping suite runs.The AUS2200 suite has the ability to manage this, specifically for soil moisture fields. The suite can be configured to carry over soil moisture data between 24-hour restart runs. This is controlled by a combination of the `create_soil_moisture_ancil` task, and the input into the `aus2200_d0198_RA3_um_recon` task. To enable soil moisture carry-over, set `SOIL_MOISTURE_CARRYOVER` to true, and set `SPINUP_HOURS` to an integer values 
+![soil moisture carryover](../Images/AUS2200/soil-moisture-carryover.PNG).
+
+Spin up hours is only used in restart runs, and determines how many hours of the model run are estimated to be required to 'spin-up' the atmosphere model. Typically, the model is restarted every 24 hours, and run for `24 + SPINUP_HOURS` model hours,  with the first `SPINUP_HOURS` discarded. In order for the soil moisture carryover to work, the suites must adhere to a strict naming convention that allows the `create_soil_moisture_ancil` task to derive the name of the previous restart suite. The convention is as follows:
+```{note}
```

Comment:
```suggestion
```{admonition} Naming convention
```

Reply:
This will create a box like for the note but with a title, I think something like this, feel free to use a better title will work better for this case, as you're referring to it beforehand it's more an example/clarification than a "note"